protos/bottle/fulfillment/v1/tax_service.proto

```diff
@@ -0,0 +1,20 @@
+syntax = "proto3";
+
+import "bottle/fulfillment/v1/order.proto";
+import "bottle/fulfillment/v1/tax_line.proto";
+
+package bottle.fulfillment.v1;
+
+message TaxListRequest {
+  string request_id = 1;
+  bottle.fulfillment.v1.Order order = 2;
+}
+
+message TaxListResponse {
+  string request_id = 1;
+  repeated bottle.fulfillment.v1.TaxLine tax_line = 2;
+}
+
+service TaxService {
+  rpc TaxList(TaxListRequest) returns (TaxListResponse);
```

Comment:
Super small service that should be all we need. Plus it's easy to replace if we need to. Also by making it a bottle, none of the new code should require database persistence and all the issues that come with that when trying to preview an order before it's created.

Reply:
I'd like to return the Order populated with tax lines rather than the Tax Lines themselves. There will almost certainly come a time when Avalara provides additional information (like using the document manager would manage exemptions for us, we'd want to flag that on the order as it comes back).

Reply:
I think it would be better to just add those documents to `TaxListResponse` (probably rename the function then too). Either way we will need to access the specific data unless we cast _the whole order_ with no function processing, which will be super hard to debug.

Basically, it would be better to have the response more structured than have the tax service modify any part of the order.

Reply:
I'm happy to defer to you here but I would encourage us to rethink this since it deviates from how we've built our few other services. Just yesterday we discussed builds including a whole order message inside them, even if it's just got an id populated. It's also been mentioned in relation to updating LossPrevention to return an Order vs flags once Hal is on Rabbit.

Reply:
I'll switch it to order but only the id and tax lines will be populated :+1: